### ISSUE TITLE
The instructions to make Rustdesk work with SELinux don't work on Fedora 39

### ISSUE THREAD

**Issue by LostinSpacetime** [rustdesk/doc.rustdesk.com#338]
### Bug Description

The solutions offered here: [https://rustdesk.com/docs/en/client/linux/selinux/](https://rustdesk.com/docs/en/client/linux/selinux/) are not working.

Tested on Fedora 39 with Wayland.

### How to Reproduce

- Install Rustdesk nightly.
- Follow one of the Instructions to add the necessary SELinux policies
- restart computer
- try to remotely connect to computer

### Expected Behavior

-connection works and one can controll the remote device

instead:

-connection works, but input devices (mouse and keyboard) don't

### Operating system(s) on local side and remote side

Fedora 39

### RustDesk Version(s) on local side and remote side

1.2.4 nightly

### Screenshots

no screenshot required

### Additional Context

I also tested this workaround without success: https://github.com/rustdesk/rustdesk/issues/3565#issuecomment-1704210322

**Comment by rpgdev** [comment]
Thank you for reporting this. You just saved me from starting that convoluted process.. didn't look pretty.

**Comment by LostinSpacetime** [comment]
> Thank you for reporting this. You just saved me from starting that convoluted process.. didn't look pretty.

The easiest way to make it work on Fedora is to disable SELinux.

**Comment by rpgdev** [comment]
> > Thank you for reporting this. You just saved me from starting that convoluted process.. didn't look pretty.
> 
> The easiest way to make it work on Fedora is to disable SELinux.

Yeah, no. I'll just keep autostarting the app on login. Not like I log in that often.

**Comment by elhananjair** [comment]
> > Thank you for reporting this. You just saved me from starting that convoluted process.. didn't look pretty.
> 
> The easiest way to make it work on Fedora is to disable SELinux.

Disabling SELinux is not recommended, I followed this guide and it worked for me, https://www.axelrafn.org/blog/selinux-rustdesk/
